cmd/kubeadm/app/apis/kubeadm/validation/validation.go

```diff
@@ -638,3 +649,15 @@ func ValidateSocketPath(socket string, fldPath *field.Path) field.ErrorList {
 
 	return allErrs
 }
+
+// ValidateImageRepository validates format of image repository
+func ValidateImageRepository(imageRepository string, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	dummyImage := fmt.Sprintf("%s/%s:%s", imageRepository, "dummy", "1.0")
```

Comment:
i think we should change the word "dummy" to just "image".
i remember some complains about it.